bootcamp/users/views.py

```diff
@@ -7,14 +7,21 @@
 from django.views.generic import DetailView, ListView, RedirectView, UpdateView
 from django.conf import settings as django_settings
 from .models import User
+from ..news.models import News
 
 
 class UserDetailView(LoginRequiredMixin, DetailView):
     model = User
+    paginate_by = 20
```

Comment:
This is a CBV implementation for a DetailView, what is the need to incorporate pagination?